components/vf-banner-elixir/vf-banner-elixir.react.js

```diff
@@ -0,0 +1,39 @@
+// vf-banner-elixir for React
+// See vf-extensions-react for usage guidance
+// We use vanilla JS templates for react for compatibility with create react app
+// ---
+import React from "react";
+import Fragment from "react-dom-fragment";
+// eslint-disable-next-line no-unused-vars
+import VfBanner-ElixirTemplate from "./vf-banner-elixir.precompiled.js"; // import templates before the nunjucks env
+import { vfNunjucksEnv } from "@visual-framework/vf-extensions-react/vf-extensions-react.js";
+
+// any JS actions needed on component insertion
+class VfBanner-ElixirCallback extends React.Component {
```

Comment:
I don't think anything with `-` is a valid name for a class, it wouldn't compile.

Reply:
Ah, thanks there was a bug in the component generator (since fixed)

Reply:
Pushed a fix